buildchain/requirements-Darwin.txt

```diff
@@ -0,0 +1,9 @@
+#
```

Comment:
Ugh :-( How is one with a Linux machine supposed to update this file now?

Reply:
Yup, Mac people are responsible to update it if they want it to work for them (I asked Carlito to generate this file) :S

I'm not that happy with the current solution but I didn't find a better way with `pip-compile` :-(
What I explored was:
- let people run `pip-compile` themselves: not good, they have to install `pip-tool` (an extra-dep, and extra-step) and we lose the reproducible build feature.
- using environment markers (as supported by pip): can't make it work with `pip-compile` though (it evaluate them at "compile" time instead of copying them to the generated `requirements.txt` so "compiling" on Linux force the use of `inotify` => issue is not solved).